src/renderer/contexts/ExecutionContext.tsx

```diff
@@ -478,7 +479,10 @@ export const ExecutionProvider = memo(({ children }: React.PropsWithChildren<{}>
                 });
             }
         } catch (err: unknown) {
-            if (!(err instanceof DOMException && err.name === 'AbortError')) {
+            if (
+                !(err instanceof DOMException && err.name === 'AbortError') &&
```

Comment:
Is this even needed anymore? I don't think so, but just to be sure, I think it would be good if `fetchJson` automatically converted these `DOMException`s to our `CancelError`, so we only have to deal with our own errors.

When moving this to `Backend.ts`, be careful to check whether `DOMException` is defined, because it's a web-only class. So the condition needs to be something like `if (typeof DOMException !== "undefined" && err instanceof DOMException && err.name === 'AbortError')`.

Reply:
You know, I think we actually don't need that part at all, like I think the axios cancel completely replaces this.

At least, I cant get that condition to trigger now